src/NBomber/Contracts.fs

```diff
@@ -29,10 +29,30 @@ type TestFlow = {
 type Scenario = {
     ScenarioName: string
     TestInit: IStep option
-    TestFlows: TestFlow[]    
+    TestFlows: TestFlow[]   
     Duration: TimeSpan
+    Assertions: AssertionScope[]
 }
 
+type AssertionInfo = {
```

Comment:
it seems to me that a better name is AssertionStats, isn't?

Reply:
AssertionInfo -> AssertionStats

Reply:
AssertionInfo -> AssertionStats